### PR TITLE
DAOS-7321 Test: Adding Partial strip cases for rebuild.

### PR DESCRIPTION
**Pull request by ravalsam**
Adding Partial update IO for offline rebuild and disabled rebuild existing test case.

Quick-Functional: true
Test-tag-hw-large: pr ec_offline_rebuild_array ec_disabled_rebuild_array

Signed-off-by: Samir Raval <samir.raval@intel.com>

### CODE REVIEW

**Comment by ravalsam** [comment]
ec_offline_rebuild.yaml is failing for partial writes because of defect DAOS-8502.

ec_online_rebuild.yaml is skipped because of exiting defect DAOS-7378.

ec_rebuild_disabled.py test passed and result can be found from https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-6625/3/artifact/Functional%20Hardware%20Large/erasurecode/ec_rebuild_disabled.py/

**Review by rpadma2 — APPROVED** [review]
LGTM.

File: src/tests/ftest/erasurecode/ec_offline_rebuild.yaml [hunk 1]
```diff
@@ -71,8 +71,9 @@ ior:
   repetitions: 1
   chunk_block_transfer_sizes:
    # [ChunkSize, BlocksSize, TransferSize]
-   - [32M, 128M, 8M]       #Full Striped
-   #- [32M, 128M, 2K]       #Patrtial Striped
+   - [32M, 128M, 8M]       # Full Striped
+   # Remove this line when DAOS-8502 defect is resolved.
+   #- [32M, 16M, 2K]       # Partial Striped
```
Comment (saurabhtandan) [inline src/tests/ftest/erasurecode/ec_offline_rebuild.yaml:76]: Is this accidentally got left commented or is supposed to stay commented?

**Review by saurabhtandan — APPROVED** [review]
LGTM

File: src/tests/ftest/erasurecode/ec_offline_rebuild.yaml (hunk 1 above)
Comment (ravalsam) [reply src/tests/ftest/erasurecode/ec_offline_rebuild.yaml:76]: It is commented until DAOS-8502 is resolved.

File: src/tests/ftest/erasurecode/ec_offline_rebuild.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ hosts:
   test_clients:
     - client-A
     - client-B
-timeout: 900
+timeout: 2500
```
Comment (johannlombardi) [inline src/tests/ftest/erasurecode/ec_offline_rebuild.yaml:25]: That's a very long time out ....

Comment (ravalsam) [reply src/tests/ftest/erasurecode/ec_offline_rebuild.yaml:25]: With partial IO some tests are taking 2384 seconds.

https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-6625/3/artifact/Functional%20Hardware%20Large/erasurecode/ec_rebuild_disabled.py/results.html

**Pull request merged**